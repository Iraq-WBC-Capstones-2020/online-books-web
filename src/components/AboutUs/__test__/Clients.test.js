import React from 'react';
import renderer from 'react-test-renderer';
import Detailsclient from './../Clients';

test('Clients snapshot', () => {
  const tree = renderer.create(<Detailsclient />).toJSON();
  expect(tree).toMatchSnapshot();
});
