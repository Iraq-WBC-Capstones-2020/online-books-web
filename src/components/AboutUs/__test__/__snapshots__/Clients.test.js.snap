// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clients snapshot 1`] = `
<div
  className="container"
  data-test="container"
>
  <div
    className="text-center py-3"
  >
    <h1
      className="text-3xl font-medium inline border-b border-gray-400 px-5"
    >
      clientPage.clients
    </h1>
  </div>
  <div
    className="row m-5"
    data-test="row"
  >
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="myob company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-1.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="Belimo company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-2.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="life groups"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-3.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="lily company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-4.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="citrus company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-5.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="trusly"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-6.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="oldendorff company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-7.png"
        />
      </div>
    </div>
    <div
      className="col-12 col-xs-6 col-sm-6 col-md-4 col-lg-3 m-0 p-0 border border-solid border-4 border-gray-200 "
      data-test="col"
    >
      <div
        className="flex justify-center"
      >
        <img
          alt="grabuo company"
          className="animate transition duration-500 ease-in-out   transform hover:-translate-y-1 hover:scale-110 w-1/2 m-2"
          src="client-8.png"
        />
      </div>
    </div>
  </div>
</div>
`;
